pkg/dom0-ztools/Dockerfile

```diff
@@ -4,29 +4,17 @@ ENV BUILD_PKGS git patch ca-certificates util-linux build-base gettext-dev libti
 ENV PKGS ca-certificates util-linux libintl libuuid libtirpc libblkid libcrypto1.1 zlib
 RUN eve-alpine-deploy.sh
 
-COPY /patches /
-
 # should be aligned with kernel
 #  * ZFS on Linux
-ENV ZFS_VERSION=2.1.2
+ENV ZFS_VERSION=2.2.2
 ENV ZFS_COMMIT=zfs-${ZFS_VERSION}
 ENV ZFS_REPO=https://github.com/openzfs/zfs
-ENV ZFS_PATCH_DIR=/patches-zfs-"${ZFS_VERSION}"
 
 WORKDIR /tmp/zfs
 
 ADD ${ZFS_REPO}/tarball/${ZFS_COMMIT}/ zfs.tgz
 RUN tar -zxvf zfs.tgz  --strip-components=1 && \
```

Comment:
@andrewd-zededa since you are touching here I would like to propose an improvement apart from your required changes: Eliminate the additional step of extracting the tarball.... we can just simply fetch directly from github using the ADD command, like this:

```diff
diff --git a/pkg/dom0-ztools/Dockerfile b/pkg/dom0-ztools/Dockerfile
index dcd7c86a6..3ba4a396c 100644
--- a/pkg/dom0-ztools/Dockerfile
+++ b/pkg/dom0-ztools/Dockerfile
@@ -10,14 +10,12 @@ COPY /patches /
 #  * ZFS on Linux
 ENV ZFS_VERSION=2.1.2
 ENV ZFS_COMMIT=zfs-${ZFS_VERSION}
-ENV ZFS_REPO=https://github.com/openzfs/zfs
+ENV ZFS_REPO=https://github.com/openzfs/zfs.git
 ENV ZFS_PATCH_DIR=/patches-zfs-"${ZFS_VERSION}"
 
 WORKDIR /tmp/zfs
+ADD ${ZFS_REPO}#${ZFS_COMMIT} /tmp/zfs
 
-ADD ${ZFS_REPO}/tarball/${ZFS_COMMIT}/ zfs.tgz
-RUN tar -zxvf zfs.tgz  --strip-components=1 && \
-    rm zfs.tgz
```

Reply:
Sure I'll make the change.

Reply:
Done